eversql/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
If no metrics are being submitted, this file can be deleted. You'll also want to remove the `integration` block (lines 34 through 48) in the `manifest.json` file as well.